feat: add documentation as readme.md file for each SDK plugins #235 (#240)
AmadeusITGroup/otter: packages/@ama-sdk/core/src/plugins/exception/readme.md

## Exception

> Plugin to fire an exception on wrong response / data according to response status or custom criteria.

Reviewer: ```suggestion
Plugin to trigger an exception in case of wrong response / data based on the response status or custom criteria.
```
